### PR TITLE
Features/bump to 0.9.13 + nft

### PR DESCRIPTION
**Pull request by willeslau**
Bumped to 0.9.13 and added nft

### CODE REVIEW

File: substrate-node-example/runtime/src/lib.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
-// use orml_nft;
+use orml_nft;
```
Comment (clearloop) [inline substrate-node-example/runtime/src/lib.rs:12]: what exactly this PR do? no difference from branch main if merge main into this branch

**Review by ivanceras — APPROVED** [review]

**Review by clearloop — CHANGES_REQUESTED** [review]
bugs in this PR have already been fixed in https://github.com/ChainSafe/filecoindot/pull/119/commits/3119928e1822cfa1184a7fa4376786eb90a637e2 (#119), guess this PR wants to configure `orml_nft` into the node-template's runtime? this have already done in the `main` branch also

File: substrate-node-example/runtime/src/lib.rs (hunk 1 above)
Comment (willeslau) [reply substrate-node-example/runtime/src/lib.rs:12]: strange... I added the orml_nft configs into runtime. But seems already merged into main...

Comment (clearloop) [reply substrate-node-example/runtime/src/lib.rs:12]: emmm, I fixed and merged that yesterday =.=

**Pull request closed without merging**